README.md

```diff
@@ -1 +1,20 @@
-# Hello_Microverse-
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Hello Microverse
+
+> My first repository at Microverse
+
+## Built With
+
+- HTML
+- CSS
+
+👤 **Zuhra "Hashimi"**
+
+- GitHub: [@ZuhRa-HashiMi](https://github.com/ZuhRa-HashiMi)
+- Twitter: [@ZuhraHashimi4](https://twitter.com/ZuhraHashimi4)
+- LinkedIn: [zuhra-hashimi](https://www.linkedin.com/in/zuhra-hashimi-601966214/)
+
+## 📝 License
+
+This project is [MIT](LICENSE.md) licensed.
```

Comment:
- [ ] Please make sure that you follow the readme template from [here](https://github.com/microverseinc/readme-template) with all applicable headers for this project, like the `Built With`, `Getting Started`, `Acknowledgments`, `Contributions` & `PreRequisites`. 
- [ ] Your License link is not working because here you have linked to a file called `LICENSE.md` and the license in your repo is only called `LICENSE` . Kindly try to fix this broken link